apps/web/components/PlazaCard/PlazaCard.tsx

```
@@ -0,0 +1,162 @@
+  }
+
+  const renderDetails = () => {
+    if (!detailLines?.length) {
+      return null
+    }
+
+    return (
+      <Box marginTop={2}>
+        <Stack space={1}>
+          {detailLines?.map((d) => (
+            <Box display="flex" flexDirection={'row'} alignItems="center">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)